Signal/src/call/PeerConnectionClient.swift

```diff
@@ -334,181 +377,220 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
 
     public func createOffer() -> Promise<HardenedRTCSessionDescription> {
         SwiftAssertIsOnMainThread(#function)
+        let handle = self.handle
+        let (promise, fulfill, reject) = Promise<HardenedRTCSessionDescription>.pending()
+        let completion: ((RTCSessionDescription?, Error?) -> Void) = { [weak self] (sdp, error) in
+            guard let strongSelf = handle.get() else { return }
+            strongSelf.assertOnSignalingQueue()
+            guard strongSelf.peerConnection != nil else {
```

Comment:
I guess it's worth noting that despite all of the whitespace changes that came from unwrapping previously nested blocks, very little has changed in the business logic here.